addons/mail/static/src/js/tour.js

```diff
@@ -32,8 +32,8 @@ tour.register('mail_tour', {
     },
 }, {
     trigger: ".ui-menu-item > a",
-    auto: true,
-    in_modal: false,
+    content: _t("Create a public or private channel."),
```

Comment:
I'm not expert in translations but I believe something has to be added in pot/po files when adding new strings in stable.

Reply:
Done!